microsoft/testsuites/xdp/xdptools.py

```diff
@@ -82,6 +81,13 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         super()._initialize(*args, **kwargs)
         self._command: PurePath = PurePath(self._default_command)
         self._gro_lro_settings: Dict[str, DeviceGroLroSettings] = {}
+        # v1.4.1 requires clang-11
+        self._xdp_tools_tag = "v1.4.1"
```

Comment:
use the version class to parse and compatible with higher versions.

Reply:
@squirrelsc , sorry I did not understand your comment.
Are you suggesting to use Version class for the git tag?

or for are you referring to this comparison `self.node.os.information.version == "18.4.0"`

Reply:
Oh, yes, I misunderstood it. The logic on os version should be like 'self.node.os.information.version <= "18.4.0"`

Reply:
The reason for not using <= is because during the dependency installation, the below code already exists

```
if self.node.os.information.version < "18.4.0":
                raise UnsupportedDistroException(self.node.os)
```

Reply:
If so, you can add comments to explain why only one version specified here. Comments is to help others understand.